components/atoms/License.tsx

```diff
@@ -0,0 +1,39 @@
+import { styled } from "@mui/material/styles";
+import { grey } from "@mui/material/colors";
+import Link from "@mui/material/Link";
+import type { ContentSchema } from "$server/models/content";
+import licenses from "$utils/licenses";
+
+const Text = styled("p")({
+  color: grey[700],
+  fontSize: 12,
+});
+
+type Props = {
+  license: ContentSchema["license"];
+};
+
+export default function License({ license }: Props) {
+  if (!Object.keys(licenses).includes(license))
+    return <Text>この作品は{license} ライセンスの下に提供されています</Text>;
```

Comment:
和文と英単語・記号の間にスペースは入れるか否か、文としてはどちらか一方に統一してあったほうがより印象よくなる気がしました。

Reply:
https://github.com/mozilla-japan/translation/wiki/Editorial-Guideline にあるようなガイドラインを定め表記を統一することが理想的なのでしょうが、

この箇所に限って言えば、以下のような既存の箇所を踏襲し

英字と日本語の境界にはスペースを入れるのが適当かと思いました

https://github.com/npocccties/chibichilo/blob/a44269a7492640a65ea9e85fd7fc0126b945e5fa/components/organisms/AuthorsInput.tsx#L106

```suggestion
    return <Text>この作品は {license} ライセンスの下に提供されています</Text>;
```